### PR TITLE
Add deamon-logger-extra-bundle recipe

### PR DESCRIPTION
**Pull request by Deamon**
Add manifest for logger extra bundle

| Q             | A
| ---------- | ---
| License    | MIT

### CODE REVIEW

**Comment by Deamon** [comment]
Well, I do not intent to provide a recipe for version previous to 4.0 so the travis failed job isn't relevant.

Is it an obligation to do a 3.4 recipe ?

File: deamon/logger-extra-bundle/4.0/config/packages/deamon_logger_extra.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# see https://github.com/FrDeamon/logger-extra-bundle/blob/master/README.md#config-reference
+deamon_logger_extra:
+    # the application name
+    application: ~
+    # the handler to update
+    handlers: 'main'
+    # configuration for all optionnal fields
```
Comment (Nyholm) [inline deamon/logger-extra-bundle/4.0/config/packages/deamon_logger_extra.yaml]: Typo `s/optionnal/optional`

Comment (Deamon) [reply deamon/logger-extra-bundle/4.0/config/packages/deamon_logger_extra.yaml]: thanks for the feedback. it is now fix.

**Comment by Deamon** [comment]
thank you for your answer @Nyholm, I fixed your feedback today.

**Comment by Deamon** [comment]
Hi @Nyholm, is it ok for you now ?

**Pull request merged**